pysoa/test/server.py

```diff
@@ -7,9 +7,8 @@
 import os
 import unittest
 
-import six
-
 from pysoa.client import Client
+from pysoa.test.assertions import raises_call_action_error, raises_field_errors, raises_error_codes
```

Comment:
nit, line too long?

Reply:
This must be a multi-line import. Please run `isort` and resolve all issues (except one in an unrelated file where it tries to move the location of a comment within imports, which is an `isort` bug).